src/Compilers/CSharp/Test/Symbol/Symbols/ModuleInitializersTests.cs

```diff
@@ -208,7 +208,16 @@ class C
 
 namespace System.Runtime.CompilerServices { class ModuleInitializerAttribute : System.Attribute { } }
 ";
-            CompileAndVerify(source, parseOptions: s_parseOptions);
+            CompileAndVerify(
+                source,
+                parseOptions: s_parseOptions,
+                options: TestOptions.DebugDll.WithMetadataImportOptions(MetadataImportOptions.All),
+                symbolValidator: module =>
+                {
+                    Assert.Equal(MetadataImportOptions.All, ((PEModuleSymbol)module).ImportOptions);
+                    var rootModuleType = module.ContainingAssembly.GetTypeByMetadataName("<Module>");
+                    Assert.NotNull(rootModuleType.GetMember(".cctor"));
```

Comment:
Huh, it just occurred to me that we don't have a straightforward way to verify the IL (a la CompilationVerifier.VerifyIL).

Reply:
Do you think this is something I should look into for this PR?

Reply:
It seems like not enough reward for the effort.